src/views/App/index.tsx

```diff
@@ -31,8 +37,14 @@ import UserInfoWrapper from 'views/UserInfoWrapper';
 import './index.scss';
 
 const AppRoutes = () => {
+  const location = useLocation();
   const flags = useFlags();
 
+  // Scroll to top
+  useLayoutEffect(() => {
```

Comment:
Why `useLayoutEffect` over `useEffect`?

I chose `useLayoutEffect` because it executes synchronously with the first render of the component. `useEffect` executes after the first render. I think it's nice to use `useLayoutEffect` so users don't see a quick flash of a page changing scroll position.